HHS/Head-Start-TTADP
# [Prod] Fix bugs with date filters

## Description of change
The start date after and start date before grant scopes were excluding too many grants. The scope has been modified to return all the recipients with grants active during the time frame. An end date scope was added to duplicate the start date functionality so it would show grants during the same time frame (I think that makes sense).

Also fixed an error where date "is" was missing a default value (so if you selected Date started and then is and didn't select "Last thirty days" specifically, it would be registered as an invalid date)

## How to test
View the overview widget. Make sure start date before & after and end date before & after return sensible numbers. Also double check the scopes and make sure those make logical sense.

On a date started or date ended filter, select "is" and then hit apply without touching the final combobox (last thirty days/year to date). You should see a filter applied.

## Issue(s)

- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-697
- https://ocio-jira.acf.hhs.gov/browse/TTAHUB-699

## Checklists

### Every PR


- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
